File: utils/src/main/java/com/xiong/utils/sp/LogUtils.java
package com.xiong.utils.sp;

/**
 * 类描述：Log工具类
 */

import android.util.Log;

public class LogUtils {

    private static final String TAG = "MyApp"; // 这里可以设置默认的标签名


    // 日志级别：Verbose
    public static void v(String message) {
        Log.v(TAG, message);
    }


    // 日志级别：Debug
    public static void d(String message) {
        Log.d(TAG, message);
    }


    // 日志级别：Info
    public static void i(String message) {
        Log.i(TAG, message);
    }


    // 日志级别：Warning
    public static void w(String message) {
        Log.w(TAG, message);
    }


    // 日志级别：Error
    public static void e(String message) {
        Log.e(TAG, message);
    }
}

File: utils/src/main/java/com/xiong/utils/sp/LogUtils.kt
package com.xiong.utils.sp

import android.util.Log

/**
 * 类描述：Log工具类
 */
object LogUtils {
    private const val TAG = "MyApp" // 这里可以设置默认的标签名

    // 日志级别：Verbose
    fun v(message: String?) {
        Log.v(TAG, message!!)
    }

    // 日志级别：Debug
    fun d(message: String?) {
        Log.d(TAG, message!!)
    }

    // 日志级别：Info
    fun i(message: String?) {
        Log.i(TAG, message!!)
    }

    // 日志级别：Warning
    fun w(message: String?) {
        Log.w(TAG, message!!)
    }

    // 日志级别：Error
    fun e(message: String?) {
        Log.e(TAG, message!!)
    }
}
